app/models/manageiq/providers/autosde/storage_manager/event_catcher/runner.rb

```diff
@@ -31,9 +31,14 @@ def queue_event(event)
 
   private
 
+  def prepend_fixed_to_event_type(event)
+    is_fixed = event.fixed == 'yes' ? 'fixed' : 'not fixed'
+    "#{is_fixed}: #{event.event_type}"
```

Comment:
What are the event types?  Does it make sense to have `not fixed` in front of all of them?
E.g. I see `AUTOSDE_instance_power_on` I don't think that `not fixed: AUTOSDE_instance_power_on` makes sense.

How about
```suggestion
    event.fixed == 'yes' ? "#{event.event_type}_fixed" : event.event_type
```

Reply:
I agree, the thing is appending `fixed` makes it invisible in the chart legend.

`fixed` appended:
![image](https://user-images.githubusercontent.com/106743023/187367174-711c2882-0d1b-4ea9-9ae5-fb45f3f17456.png)

'fixed' prepended (ignore `unfixed`, I agree it's redundant):
![image](https://user-images.githubusercontent.com/106743023/187367623-861adfb9-bba6-4b33-8152-f5d1b1c99347.png)
 
I know this problem belongs to a different repo/PR: https://github.com/ManageIQ/manageiq/pull/22081, 
but that's the whole reason for this change..
If I could control react carbon graph options through the yaml I would try to sort out the legend, but it's unclear how to do so (as I also wrote on Gitter).